### ISSUE TITLE
[dynamo+ddp+symbolic-shapes] Issue Tracker

### ISSUE THREAD

**Issue by wconstab** [pytorch/pytorch#93618]
we should support these systems working togehter.  i think they should compose, but it requires some testing.

- [ ] huggingface accelerate nlp_model crashes ([repro cmd](https://gist.github.com/wconstab/12ce82ceded7a3ee4748b24cc5ec502e#file-repro-instructions), [log](https://gist.github.com/wconstab/12ce82ceded7a3ee4748b24cc5ec502e#file-symbolic_nlp_example-log))
- [ ] torchbench hf_Bert is slow with symbolic-shapes (`python benchmarks/dynamo/distributed.py --torchbench_model hf_Bert --ddp --dynamo inductor` on symbolic-shapes branch)

cc @ezyang @soumith @msaroufim @ngimel @bdhirsh

**Comment by ezyang** [comment]
This is not a priority for symbolic shapes right now. Please holler if you think it should be.

**Comment by wconstab** [comment]
yea, it isn't the highest prio, but i think it should be revisited once more basic stuff is done

**Comment by ydwu4** [comment]
I couldn't repro the huggingface accelearete nlp_model. For the hf_Bert slow issue I was able to run the command and got following output:
```
(repro) ~/local/pytorch$ python benchmarks/dynamo/distributed.py --torchbench_model hf_Bert --ddp --dynamo inductor
NCCL version 2.19.3+cuda12.1
[rank0]:[2023-11-29 10:59:31,856] [0/0] torch._dynamo.variables.torch: [WARNING] Profiler function <class 'torch.autograd.profiler.record_function'> will be ignored
[rank0]:[2023-11-29 10:59:38,703] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard Eq(s0, 512) == True, this could result in accuracy problems
[rank0]:[2023-11-29 10:59:38,726] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard Eq(s1, 4) == True, this could result in accuracy problems
[rank0]:[2023-11-29 10:59:43,908] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard Eq(s2, 3072) == True, this could result in accuracy problems
[rank0]:[2023-11-29 10:59:43,919] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard Eq(s3, 768) == True, this could result in accuracy problems
[rank0]:[2023-11-29 11:00:07,169] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard Eq(s2, 768) == True, this could result in accuracy problems
[rank0]:[2023-11-29 11:00:07,590] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard 768*s0*s1 < 2147483648 == True, this could result in accuracy problems
[rank0]:[2023-11-29 11:00:07,597] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard 768*s1*(s0 - 1) + 768*s1 < 2147483648 == True, this could result in accuracy problems
[rank0]:[2023-11-29 11:00:07,602] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard s1*(s0 - 1) + s1 < 2147483648 == True, this could result in accuracy problems
[rank0]:[2023-11-29 11:00:08,449] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard 12288*(((s0*s1 + 15)//16)) < 2147483648 == True, this could result in accuracy problems
[rank0]:[2023-11-29 11:00:09,043] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard 30522*s0*s1 < 2147483648 == True, this could result in accuracy problems
[rank0]:[2023-11-29 11:00:09,049] torch.fx.experimental.symbolic_shapes: [WARNING] Ignored guard 30522*s1*(s0 - 1) + 30522*s1 < 2147483648 == True, this could result in accuracy problems
mean latency 0.04021873446181416 across 10 runs
```

The latency seems reasonable to me lol. Can we close the issue in this case?

**Comment by ydwu4** [comment]
Closed as priority is low but feel free to re-open.
